tests/cli/commands/test_task_command.py

```diff
@@ -106,7 +106,41 @@ def test_test_with_existing_dag_run(self):
             task_command.task_test(args)
 
         # Check that prints, and log messages, are shown
-        assert f"Marking task as SUCCESS. dag_id={self.dag_id}, task_id={task_id}" in stdout.getvalue()
+        assert f"Marking task as SUCCESS. dag_id={dag_id}, task_id={task_id}" in stdout.getvalue()
```

Comment:
```suggestion
        assert f"Marking task as SUCCESS. dag_id={self.dag_id}, task_id={task_id}" in stdout.getvalue()
```
Any reason for the change?

Reply:
I am pretty sure that was automatically done by me re-basing. I definitely didn't do that by hand at least :p